datalake/src/main/resources/application.yml

```diff
@@ -111,8 +111,8 @@ datalake:
       ssl: false
   runtimes:
     default: "7.2.13"
-    advertised: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.14"
-    supported: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.14"
+    advertised: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.147.2.15"
```

Comment:
`'` is missing here as well at the end of the line

Reply:
Fixed here as well. Thanks